index.js

```diff
@@ -55,7 +55,7 @@ var Module = (function () {
      * @returns {boolean}
      */
     function isElement(obj) {
-      return typeof obj === 'object' && typeof obj.nodeName === 'string';
+      return (obj && obj.nodeType === 1) === true;
```

Comment:
Why `=== true`?  Code is expensive on the client-side.  If you _really_ need a boolean, `!!` is only two characters compared to you ` === true` nine characters.